src/servers/api-server.ts

```diff
@@ -72,6 +88,21 @@ apiServer.get("/api/rest/2.0/*", async (c) => {
     });
 });
 
+apiServer.get("/api/test/ping", async (c) => {
```

Comment:
isnt this a `tool` so lets be consistent

Reply:
done